[DOCS-11676] Adds API and env var updates (#31859)
DataDog/documentation: content/en/observability_pipelines/set_up_pipelines/_index.md

<div class="alert alert-warning">Creating pipelines using the Observability Pipelines API is in Preview. Fill out the <a href="https://www.datadoghq.com/product-preview/observability-pipelines-api-and-terraform-support/"> form</a> to request access.</div>

1. You can use Observability Pipelines API to [create a pipeline][1].

> 1. After the pipeline has been created, [install the Worker][2] to start sending logs through the pipeline.

Reviewer: This is kind of a stacked verb construction. Forces the reader to carry and connect a lot of actions.

You’ve got “is created” (passive verb phrase), then “install” (imperative), then “to start sending” (infinitive + gerund).

That’s three different verbal ideas crammed into one sentence, which makes it feel busy.

Maybe something like `After creating the pipeline, install the Worker to send logs through it.` is easier on the reader?

Author: Thanks!

> 1. After the pipeline has been created, [install the Worker][2] to start sending logs through the pipeline.

Author: ```suggestion
1. After creating the pipeline, [install the Worker][2] to send logs through it.
```
